### PR TITLE
Strip off extra quotes wrapping each link command line

### LINKED ISSUE #1088: Link stage fail if using Response file
### Description
HCC would create a temp Response File under /tmp/response-xxx.txt to take the whole command line if the length of the command line does not fit in system limit. Then the **clamp-link** would expand the Response File  and pass each command to host linker. However, linker would report a bunch of error: 

> ld: cannot find "-L/opt/rocm/hcc/bin/../lib": No such file or directory
> ld: cannot find "--rpath=/opt/rocm/hcc/bin/../lib": No such file or directory
> ld: cannot find "-ldl": No such file or directory
> ld: cannot find "-lm": No such file or directory
> ld: cannot find "-lpthread": No such file or directory
> ld: cannot find "-lhc_am": No such file or directory
> ld: cannot find "-lmcwamp": No such file or directory
> ld: cannot find "--amdgpu-target=gfx900": No such file or directory
>...
### How to reproduce
-  Compile the sample code HelloWorld.cpp to generate helloworld.o [HelloWorld.cpp.txt](https://github.com/RadeonOpenCompute/hcc/files/2987391/HelloWorld.cpp.txt)
`hipcc -o helloworld.o -c HelloWorld.cpp`
- As response file is used to hold comand line > system limit(>60000), i force hcc to generate a response file when compile this sample code by slight changing the source code, I have the response file generated attached [response-helloworld.txt](https://github.com/RadeonOpenCompute/hcc/files/2987437/response-helloworld.txt), so this step could be skipped

- Use clamp-link and Prebuilt Response File  [response-helloworld.txt](https://github.com/RadeonOpenCompute/hcc/files/2987405/response-helloworld.txt)
`/opt/rocm/hcc/bin/clamp-link \@response-helloworld.txt`
- Output , full log is [error.log](https://github.com/RadeonOpenCompute/hcc/files/2987422/error.log)
> ld: cannot find "-L/opt/rocm/hcc/bin/../lib": No such file or directory
> ld: cannot find "--rpath=/opt/rocm/hcc/bin/../lib": No such file or directory
> ld: cannot find "-ldl": No such file or directory
> ld: cannot find "-lm": No such file or directory
> ...

### PR DESCRIPTION
**Pull request by shimmerhuangAMD**
This is to fix #1088

### CODE REVIEW

File: lib/clamp-link.in [hunk 1]
```diff
@@ -290,6 +290,14 @@ do
     IFS=$'\n' read -d '' -r -a LINES < "$REAL_PATH"
     for LINE in ${LINES[@]}
     do
+      #remove the extra colons wrapping the command
```
Comment (scchan) [inline lib/clamp-link.in]: you mean removing the double-quotes, not colons?

**Review by scchan — CHANGES_REQUESTED** [review]

File: lib/clamp-link.in (hunk 1 above)
Comment (shimmerhuangAMD) [reply lib/clamp-link.in]: Correct, sorry for misleading by wrong word.
Changed the title and updated the commit with correct comments

Comment (scchan) [reply lib/clamp-link.in]: Could you update the comment in the code as well? thanks

Comment (shimmerhuangAMD) [reply lib/clamp-link.in]: Updated

**Review by scchan — APPROVED** [review]

**Pull request merged**